src/main/perl/MD.pm

```diff
@@ -307,33 +310,18 @@ then
 EOC
     foreach my $dev (@{$self->{device_list}}) {
         $dev->create_ks;
-        # Well, fdisk sucks and Anaconda is plain crap. Let's
-        # guess how we can set the partition hex code
```

Comment:
Feel free to reword the comment, but this piece of code is far from obvious, and I'd really appreciate an explanation on why it's needed.